test/unit/Middleware.t.sol

```diff
@@ -2642,16 +2637,28 @@ contract MiddlewareTest is Test {
         vm.stopPrank();
     }
 
-    function testVaultToOracle() public {
+    // function testVaultToOracle() public {
```

Comment:
Let's either add a TODO if we plan to update it, or remove it if we don't need it.